client/src/components/ui/InputSelect/InputSelect.module.scss

```diff
@@ -72,27 +124,43 @@
   bottom: 0;
   right: 0;
   margin: 0;
-  padding: 2.4rem 0;
   border-radius: $border-radius-16;
   z-index: $z-index-6;
   box-shadow: $box-shadow-1;
   height: auto;
   width: 100%;
   background: $color-white;
 
-  @media #{$desktop-up} {
-    position: absolute;
-    width: 12.8rem;
-    padding: 1.6rem 0;
-    bottom: auto;
+  &.variant-- {
+    &topselect {
+      position: absolute;
+      width: 14.4rem;
+      bottom: 100%;
+      left: 50%;
+      transform: translate(-50%) !important;
+      padding: 0.8rem 0;
+    }
 
+    &overselect, &belowselect {
+      padding: 2.4rem 0;
+    }
+  }
+
+  @media #{$desktop-up} {
     &.variant-- {
+      &overselect, &underselect {
```

Comment:
`underselect` to remove

Reply:
I don't see the reason. Why?

Reply:
`underselect` is only used here.

Reply:
I don't understand. Can you please rephrase?

Reply:
There is no `underselect` variant in `InputSelect` component.

Reply:
Should be `belowselect`. Fixed.